System.Compiler/Nodes.cs

```diff
@@ -16245,6 +16245,20 @@ public RequiresList Validations
       get { return this.validations; }
       set { this.validations = value; }
     }
+
+    /// <summary>
+    /// Returns true if current contract node has at least one assertion in it
+    /// (like pres/posts etc).
```

Comment:
Assertion is a very common term in this domain. I can't understand why to duplicate the info here.